### ISSUE TITLE
Switch inputs?

### ISSUE THREAD

**Issue by erikarthur** [Teque5/avr_control#1]
Hi Teque5,

Thanks for the code.  I can get my 3650 to power on/off, control volume, and mute.  I can't get the input source to switch at all.  Is there a sequence of commands I need to send to change the input source?

Thanks,

Erik

**Comment by Teque5** [comment]
I haven't done this on my particular setup. If you look at either PDF I have in the repo, the codes for selecting particular inputs seem to be present. IE `'DVD' = [0x80, 0x70, 0xD0, 0x2F]` - I have these in my python control code also. Do these commands work or not?

I think any command that is normally sent from the remote should be accessible from the rs232 port.

**Comment by erikarthur** [comment]
Yep I looked at the PDF, tried all the input source commands, and did a bunch of Googling.  It doesn't work for me.  I wonder if I need a sequence like multiroom?  I wish there was a response from the device.  Thanks.

**Comment by Teque5** [comment]
It may be worth emailing Harmon Kardon. When I did they sent me the PDFs that I uploaded here. It might not be totally useless...

**Comment by erikarthur** [comment]
Thanks.  I’ll drop them a line.  Appreciate the help.

Erik

-- 
Erik Arthur
erik@arthurweb.org


On January 16, 2017 at 7:25:54 PM, Teque5 (notifications@github.com) wrote:

It may be worth emailing Harmon Kardon. When I did they sent me the PDFs that I uploaded here. It might not be totally useless...
